datamodel/src/main/kotlin/com/hartwig/actin/datamodel/algo/TreatmentMatch.kt

```diff
@@ -10,6 +11,22 @@ data class TreatmentMatch(
     val trialMatches: List<TrialMatch>,
     val standardOfCareMatches: List<AnnotatedTreatmentMatch>?,
     val personalizedDataAnalysis: PersonalizedDataAnalysis?,
-    val survivalPredictionsPerTreatment: Map<String, List<Double>>?,
+    val survivalPredictionsPerTreatment: Map<String, TreatmentEfficacyPrediction>?,
     val maxMolecularTestAge: LocalDate? = null
 )
+
+data class TreatmentEfficacyPrediction(
+    @JsonProperty("survival_probs")
+    val survivalProbs: List<Double>,
+
+    @JsonProperty("shap_values")
+    val shapValues: Map<String, ShapDetail>
+) {
```

Comment:
What do these brackets do actually? Do they nest the data class or something? 

I don't think I'd do it this way out of consistency perspective (don't think its wrong though)